mock_drs/database/register_mongodb.py

```diff
@@ -2,78 +2,67 @@
 
 import logging
 from typing import Dict
+import json
```

Comment:
Try to keep the following order for imports:
- use three import blocks:
  1. standard Python modules
  2. third party Python modules
  3. modules within _this_ package
- within each block, arrange modules in alphabetical order (e.g., `json`, `logging`, `typing`)